invoice.go

```diff
@@ -0,0 +1,91 @@
+package ksef
+
+import (
+	"github.com/invopop/gobl/bill"
+	"github.com/invopop/gobl/regimes/common"
+)
+
+type Inv struct {
+	CurrencyCode                       string       `xml:"KodWaluty"`
+	IssueDate                          string       `xml:"P_1"`
+	SequentialNumber                   string       `xml:"P_2"`
+	CompletionDate                     string       `xml:"P_6,omitempty"`
+	StartDate                          string       `xml:"P_6_Od,omitempty"`
+	EndDate                            string       `xml:"P_6_Do,omitempty"`
+	BasicRateNetSale                   string       `xml:"P_13_1,omitempty"`
```

Comment:
Minor suggestion here. Rather than the direct translation, it might be better to use the predefined GOBL constants:
 * `Basic` would become `Standard`
 * `FirstReduced` would become`Reduced`
 * `SecondReduced` would become `SuperReduced`

Reply:
fixed